src/main/java/org/broadinstitute/hellbender/utils/read/ReadUtils.java

```diff
@@ -868,4 +871,61 @@ public static boolean alignmentAgreesWithHeader(final SAMFileHeader header, fina
         return read.isUnmapped() || read.getStart() <= contigHeader.getSequenceLength();
     }
 
+    /**
+     * Create a common SAMFileWriter for use with GATK tools.
+     *
+     * @param outputFile - if this file has a .cram extension then a reference is required. Can not be null.
+     * @param referenceFile - the reference source to use. Can not be null if a output file has a .cram extension.
+     * @param header - header to be used for the output writer
+     * @param preSorted - if true then the records must already be sorted to match the header sort order
+     * @param createIndex - if true an index will be created for .BAM and .CRAM files
+     * @param createMD5 - if true an MD5 file will be created
+     *
+     * @return SAMFileWriter
+     */
+    public static SAMFileWriter createCommonSAMWriter(
+            final File outputFile,
+            final File referenceFile,
+            final SAMFileHeader header,
+            final boolean preSorted,
+            boolean createIndex,
+            boolean createMD5)
+    {
+        Utils.nonNull(outputFile);
+        Utils.nonNull(header);
+
+        if (createIndex && header.getSortOrder() != SAMFileHeader.SortOrder.coordinate) {
+            throw new UserException("Can't CREATE_INDEX unless sort order is coordinate");
```

Comment:
"Can't create an index" -- don't refer to the Picard arg `CREATE_INDEX` in this utility class